src/components/file-preview.vue

```diff
@@ -123,21 +158,37 @@ export default {
         },
         initOptions: {
           background: false,
-          viewMode: 0,
+          viewMode: 1,
           autoCropArea: 1,
           zoomable: false
         }
       }
     };
   },
-  watch: {
-    "image.cropRatio"(newValue) {
-      this.setAspectRatio(newValue);
-    }
-  },
   computed: {
+    componentId() {
+      return "file-preview-" + ++id;
+    },
+    moreOptions() {
+      const moreOptions = Object.assign({}, this.options);
+
+      if (this.editor) {
+        moreOptions["editor"] = {
+          text: this.$t("edit"),
+          icon: "crop_rotate"
+        };
+      }
+
+      return moreOptions;
+    },
+    titleParsed() {
+      return this.title === null ? this.value.title : this.title;
+    },
+    selectRatioTranslation() {
+      return i18n.t("select_aspect_ratio");
```

Comment:
Use `this.$t` instead

```suggestion
      return this.$t("select_aspect_ratio");
```